homeassistant/components/smart_meter_texas/__init__.py

```diff
@@ -38,8 +39,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     account = Account(username, password)
 
-    client_ssl_context = ClientSSLContext()
-    ssl_context = await client_ssl_context.get_ssl_context()
+    ssl_context = client_context()
```

Comment:
```suggestion
    ssl_context = get_default_context()
```

since you are not requesting anything special let's just request the default context (for both occurrences)